CMakeLists.txt

```diff
@@ -210,6 +210,17 @@ if(ENABLE_SHARED)
     )
     # Set the SOVERSION property of the wiredtiger library, so we can export the appropriately versioned symlinks.
     set_target_properties(wiredtiger_shared PROPERTIES SOVERSION "${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_PATCH}")
+
+    # For MacOS builds we need to generate a dSYM bundle that contains the debug symbols for the 
+    # WiredTiger library.
+    if (WT_DARWIN)
```

Comment:
Do we need to check the enabling of the diagnostics flag as well?

Reply:
I actually had that check before but @korteland suggested having the debug symbols without diagnostics in the above comments.

Reply:
Got it, it's okay with me. I was just confused after reading the wiki page (the referring enable -HAVE_DIAGNOSTIC is a requirement).

Reply:
Ah good point, I'll remove it there too then.